README.md

```diff
@@ -1,226 +1,187 @@
-        .core .o--if-js,
-        .enhanced .o--if-no-js { display: none !important; }
-    </style>
-    <!-- Add CTM check -->
-    <script>
-        var cutsTheMustard = ('querySelector' in document && 'localStorage' in window && 'addEventListener' in window);
-        if (cutsTheMustard) {
-        // Swap the 'core' class on the HTML element for an 'enhanced' one
-        // We're doing it early in the head to avoid a flash of unstyled content
-        document.documentElement.className = document.documentElement.className.replace(/\bcore\b/g, 'enhanced');
-        }
-    </script>
-    <!-- Add Polyfil service -->
-    <script src="https://cdn.polyfill.io/v1/polyfill.min.js"></script>
-    <!-- INIT and make a page request -->
-    <script>
-        function oTrackinginit() {
-            // oTracking
-            var oTracking = Origami['o-tracking'];
-            var config_data = {
-                server: 'https://spoor-api.ft.com/px.gif',
-                context: {
-                    product: 'ft.com'
-                },
-                user: {
-                    ft_session: oTracking.utils.getValueFromCookie(/FTSession=([^;]+)/)
-                }
-            }
-            // Setup
-            oTracking.init(config_data);
-            // Page
-            oTracking.page({
-                content: {
-                    asset_type: 'page'
-                }
-            });
-        }
-        if (cutsTheMustard) {
-            var o = document.createElement('script');
-            o.async = o.defer = true;
-            o.src = 'https://build.origami.ft.com/v2/bundles/js?modules=o-tracking';
-            var s = document.getElementsByTagName('script')[0];
-            if (o.hasOwnProperty('onreadystatechange')) {
-                o.onreadystatechange = function() {
-                    if (o.readyState === "loaded") {
-                        oTrackinginit();
-                    }
-                };
-            } else {
-                o.onload = oTrackinginit;
-            }
-            s.parentNode.insertBefore(o, s);
-        }
-    </script>
-</head>
-<body>
-<!-- Add fallback if browsers don't cut the mustard -->
-<div class="o-tracking o--if-no-js" data-o-component="o-tracking">
-    <div style="background:url('https://spoor-api.ft.com/px.gif?data=%7B%22category%22:%22page%22,%20%22action%22:%22view%22,%20%22system%22:%7B%22apiKey%22:%22qUb9maKfKbtpRsdp0p2J7uWxRPGJEP%22,%22source%22:%22o-tracking%22,%22version%22:%221.0.0%22%7D,%22context%22:%7B%22product%22:%22ft.com%22,%22content%22:%7B%22asset_type%22:%22page%22%7D%7D%7D');"></div>
-</div>
-<!-- Send an event -->
-<button onclick="document.body.dispatchEvent(new CustomEvent('oTracking.event', { detail: { category: 'element', action: 'click' }, bubbles: true}));">Send an event</button>
-</body>
-</html>
-```
+You can configure o-tracking to listen for `oTracking.page` and `oTracking.event` events instead of using the `page` and `event` methods.
```

Comment:
This sentenced confused me at first. It might be worth being a little more verbose:

```suggestion
Instead of calling the `page` and `event` methods `o-tracking` can be configured to fire events based on the custom DOM Events `oTracking.page` and `oTracking.event`.
```

Reply:
This is much better :-)